src/Build/Graph/GraphBuildRequestData.cs

```diff
@@ -7,6 +7,14 @@
 
 namespace Microsoft.Build.Graph
 {
+    public record GraphBuildOptions
+    {
+        /// <summary>
+        /// If false, the graph is constructed but the nodes are not built.
+        /// </summary>
+        public bool Build { get; init; } = true;
```

Comment:
I wasn't familiar with this, and it looks like it's too new for mono. Might change it to set { init; return whatever; }

Reply:
Mono will get eventually updated and until then we'll just disable it. See https://github.com/dotnet/msbuild/pull/6059